.github/workflows/post_to_mastodon.yml

```
@@ -14,7 +14,7 @@ jobs:
   post_to_mastodon:
     if: |
       github.event.pull_request.merged == true &&
```

actionlint (1.7.4)

parser did not reach end of input after parsing the expression. 1 remaining token(s) in the input: ")"

(expression)